src/DssSpell.sol

```diff
@@ -38,94 +105,224 @@ contract DssSpellAction is DssAction {
+        // Poll: https://vote.makerdao.com/polling/Qmc6Wqrc#poll-detail
         // NOTE: ignore in goerli
-        // DIN-001 - 103.16 MKR - 0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F
+
+
+        // --------- Onboard GNO to Spark ---------
+        // Poll: https://vote.makerdao.com/polling/QmXdGdxS#poll-detail
+        // Forum: https://forum.makerdao.com/t/onboarding-of-gno-to-spark/20831
+        // List of addresses: https://github.com/marsfoundation/sparklend/blob/master/script/output/5/spark-latest.json
+        {
+            // Whitelist the GNO Fig adapter
+            MedianAbstract(GNO_MEDIANIZER).kiss(SPARK_GNO_ORACLE);
+
+            // Set DAI as a borrowable asset in isolation mode
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).setBorrowableInIsolation(MCD_DAI, true);
+
+            // Add GNO
+            address token = DssExecLib.getChangelogAddress("GNO");
+            PoolConfiguratorLike.InitReserveInput[] memory input = new PoolConfiguratorLike.InitReserveInput[](1);
+            input[0] = PoolConfiguratorLike.InitReserveInput({
+                aTokenImpl: SPARK_ATOKEN_IMPL,
+                stableDebtTokenImpl: SPARK_STABLE_DEBT_TOKEN_IMPL,
+                variableDebtTokenImpl: SPARK_VARIABLE_DEBT_TOKEN_IMPL,
+                underlyingAssetDecimals: GemAbstract(token).decimals(),
+                interestRateStrategyAddress: SPARK_INTEREST_RATE_STRATEGY,      // Dummy strategy - compare to other borrow-disabled asset like sDAI
+                underlyingAsset: token,
+                treasury: SPARK_TREASURY,
+                incentivesController: address(0),
+                aTokenName: "Spark GNO",
+                aTokenSymbol: "spGNO",
+                variableDebtTokenName: "Spark Variable Debt GNO",
+                variableDebtTokenSymbol: "variableDebtGNO",
+                stableDebtTokenName: "Spark Stable Debt GNO",
+                stableDebtTokenSymbol: "stableDebtGNO",
+                params: ""
+            });
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).initReserves(input);
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).configureReserveAsCollateral({
+                asset: token, 
+                ltv: 20_00,
+                liquidationThreshold: 25_00,
+                liquidationBonus: 110_00
+            });
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).setDebtCeiling(token, 5 * MILLION * HUNDRED);
+
+            address[] memory tokens = new address[](1);
+            tokens[0] = token;
+            address[] memory oracles = new address[](1);
+            oracles[0] = SPARK_GNO_ORACLE;
+            AaveOracleLike(SPARK_AAVE_ORACLE).setAssetSources(
+                tokens,
+                oracles
+            );
+        }
+
+        // Reduce Maker Protocol GNO Debt Ceiling to Zero
+        (,,,uint256 line,) = VatLike(MCD_VAT).ilks("GNO-A");
+        DssExecLib.removeIlkFromAutoLine("GNO-A");
```

Comment:
Please follow 2023-04-26 patterns

Reply:
I prefer to leave it as it is, as it's an optimal way in terms of `dss-exec-lib` interface

Reply:
Discussed sync and agreed due to this being a single ilk DC to 0 (uncommon) the way it is done here is better for single ilk changes